src/accelerate/big_modeling.py

```diff
@@ -405,6 +405,10 @@ def dispatch_model(
         # Attaching the hook may break tied weights, so we retie them
         retie_parameters(model, tied_params)
 
+        # warn if there is any params on the meta device
+        if len(offload) > 0:
+            logging.warn("Some parameters are on the meta device device because they were offloaded to the cpu/disk.")
```

Comment:
```suggestion
            logging.warning("Some parameters are on the meta device device because they were offloaded to the cpu and/or disk.")
```

Warn has been deprecated since py 3.3, let's use warning :)

Reply:
sound good